### ISSUE TITLE
TypeError: compilation.fileDependencies.includes is not a function

### ISSUE THREAD

**Issue by RobinRadic** [prncc/antd-scss-theme-plugin#33]
Got this error
```
node_modules/antd-scss-theme-plugin/build/dist/lib/index.js:21
      if (compilation.fileDependencies && !compilation.fileDependencies.includes(theme)) {
                                                                        ^
TypeError: compilation.fileDependencies.includes is not a function
```

Ran it with debugger on, it showed me `compilation.fileDependencies` is not an array but a `Set`. 

My workaround:
```es6
export class AntdScssTheme extends AntdScssThemePlugin {

    apply(compiler) {
        const afterEmit = (compilation, callback) => {
            // Watch the theme file for changes.
            const theme = AntdScssThemePlugin.SCSS_THEME_PATH;
            if ( compilation.fileDependencies && ! compilation.fileDependencies.has(theme) ) { // modified
                compilation.fileDependencies.add(theme); // modified
            }
            callback();
        };

        // Register the callback for...
        if ( compiler.hooks ) {
            // ... webpack 4, or...
            const plugin = { name: 'AntdScssThemePlugin' };
            compiler.hooks.afterEmit.tapAsync(plugin, afterEmit);
        } else {
            // ... webpack 3.
            compiler.plugin('after-emit', afterEmit);
        }
    }
}
```

**Comment by prncc** [comment]
I think that `fileDependencies` was changed to a set from an array in Webpack 4. Which version of Webpack are you using? 

Either way, thanks for reporting; I'll get it fixed first thing.

**Comment by RobinRadic** [comment]
webpack@4.10.2

**Comment by sinisavukovic** [comment]
I'm having the same issue with `webpack@4.8.3`
@RobinRadic fix solves the issue. Thanks

**Comment by samuel-alpoim** [comment]
I'm also having this issue, @RobinRadic fix is also working here, do you have some estimation about when you will apply this code in your plugin?

**Comment by prncc** [comment]
Thanks @vieira! This fix is live as of v1.0.4. @samuel-alpoim @sinisavukovic @RobinRadic
